src/core/weightedGraph.test.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import * as Weights from "./weights";
+import {Graph} from "./graph";
+import * as WeightedGraph from "./weightedGraph";
+import * as GraphTest from "./graphTestUtil";
+
+describe("core/weightedGraph", () => {
+  function expectEqual(wg1, wg2) {
+    expect(wg1.graph.equals(wg2.graph)).toBe(true);
+    expect(wg1.weights).toEqual(wg2.weights);
+  }
+
+  describe("empty", () => {
+    it("empty produces an empty WeightedGraph", () => {
+      const {weights, graph} = WeightedGraph.empty();
+      expect(graph.equals(new Graph())).toBe(true);
+      expect(weights).toEqual(Weights.empty());
+    });
+  });
+
+  describe("toJSON/fromJSON", () => {
+    it("works for an empty WeightedGraph", () => {
+      const wg = WeightedGraph.empty();
+      const wgJSON = WeightedGraph.toJSON(wg);
+      const wg_ = WeightedGraph.fromJSON(wgJSON);
```

Comment:
nit: We generally call these `wg2` rather than `wg_`\*, which is
significantly easier to read, especially in contexts where the code is
underlined (e.g., stock Vim with `set cursorline`). Could you change
these accordingly? (In cases where “`2`” is ambiguous, we also have
precedent for `newWg`.)

\* `git grep '[^ ]_ '` shows that there is no precedent for the form
used here.

Reply:
Not sure how you're coming to that conclusion that there's no precedent for this form. Here are some pre-existing examples in the codebase.

https://github.com/sourcecred/sourcecred/blob/9784927402bf4fde8716949b71092d6dcd42d72d/src/analysis/timeline/params.test.js#L28-L32

https://github.com/sourcecred/sourcecred/blob/9784927402bf4fde8716949b71092d6dcd42d72d/src/cli/common.test.js#L78-L79

https://github.com/sourcecred/sourcecred/blob/9784927402bf4fde8716949b71092d6dcd42d72d/src/core/graph.test.js#L1371-L1379

Since this is pattern is already widespread, I don't think there's value in avoiding this particular instance in this PR. However, I'd be fine accepting a change which removes them en masse and enforces compliance automatically.

Reply:
Ah—I was at an old checkout due to some archaeology; sorry. Feel free to
defer given precedent.